bin/phpunit

```diff
@@ -3,4 +3,6 @@
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 APP_DIR="$DIR/.."
 
-$APP_DIR/vendor/phpunit/phpunit/phpunit -c $APP_DIR/phpunit.xml "$@"
+mkdir -p /tmp/circleci-test-results
+
+$APP_DIR/vendor/phpunit/phpunit/phpunit -c $APP_DIR/phpunit.xml --log-junit /tmp/circleci-test-results/junit.xml "$@"
```

Comment:
I think it would make more sense to add this to the `ci/test.sh` script instead of generating this output file by default any time `bin/phpunit` is run.